.github/workflows/tekton_task_tests.yaml

```
@@ -22,8 +22,26 @@ jobs:
           dir_names: "true"
           dir_names_max_depth: "3"
       - name: Show changed dirs
+        id: show-changed-dirs
+        shell: bash
         run: |
           echo ${{ steps.changed-dirs.outputs.all_changed_files }}
+          trustedArtifactsBasedTasks=()
+          for dir in ${{ steps.changed-dirs.outputs.all_changed_files }}
+          do
+            echo "$dir"
+            baseDir=$(basename "${dir}")
+            taskFile="${dir}/${baseDir}.yaml"
+            if [ -f "${taskFile}" ]; then
+              ociStageParam=$(yq '.spec.params[] | select(.name == "ociStorage") | .name' "${taskFile}")
+              if [ ! -z "${ociStageParam}" ]; then
```

For the opposite of `-z` there is `-n`:
```suggestion
              if [ -n "${ociStageParam}" ]; then
```

---

fixed